pkg/boot/boot.go

```diff
@@ -8,25 +8,33 @@ package boot
 
 import (
 	"fmt"
+	"os"
+	"path/filepath"
 
 	"github.com/u-root/u-root/pkg/boot/kexec"
 	"github.com/u-root/uio/ulog"
 )
 
+// DefaultLinuxImageInfoFile is the default file name in tmp directory to write loaded LinuxImage info to.
+const DefaultLinuxImageInfoFile = "loadedLinuxImageInfo"
+
 // LoadOption is an optional parameter to Load.
 type LoadOption func(*loadOptions)
 
 type loadOptions struct {
 	logger        ulog.Logger
 	verbose       bool
 	callKexecLoad bool
+	// loadedInfoFile specifies where to writes loaded linuximage info to.
+	loadedInfoFile string
```

Comment:
I feel like "loadedInfoFile" is a name that only makes sense to people who have read the code. "configFile" in the struct? "linux_image_config_file.json" as the default file name? something like that?

you're basically inventing another boot entry format like GRUB or BLS

Reply:
sure, I'll use `linux_image_cfg.json` to be shorter